code/modules/virus2/effect.dm

```diff
@@ -1094,27 +1094,27 @@ datum/disease2/effect/lubefoot/deactivate(var/mob/living/carbon/mob)
 	name = "Gregarious Impetus"
 	stage = 4
 	max_chance = 25
-	max_multiplier = 2
+	max_multiplier = 4
 
 /datum/disease2/effect/gregarious/activate(var/mob/living/carbon/mob)
 	var/others_count = 0
 	for(var/mob/living/carbon/m in oview(5, mob))
 		if (airborne_can_reach(mob.loc, m.loc, 9)) // Apparently mobs physically block airborne viruses
 			others_count += 1
-	if (others_count > max_multiplier)
+	if (others_count >= multiplier)
 		to_chat(mob, "<span class='notice'>A friendly sensation is satisfied with how many are near you - for now.</span>")
 		mob.adjustBrainLoss(-multiplier)
-		mob.reagents.add_reagent(PARACETAMOL, multiplier) // The hug chemical for kicks
-		if (multiplier < 5)
-			multiplier += 0.1
+		mob.reagents.add_reagent(PARACETAMOL, multiplier) // The hug chemical
+		if (multiplier < 4)
```

Comment:
It ticks up though. If you're surrounded with people the virus gets greedier and demands more of them, if you have no one around you the virus tempers expectations and demands less of them.

If you have the max multiplier it'll never tick up as the virus can only get so greedy, but if you get any less than 4 people around you the multiplier ticks down.